NSG Service Tags: Fix AzureTrafficManager tag description

ryanbor opened the pull request:
Modified the Azure Traffic Manager tag to properly reflect the behavior of the tag. It isn't for the Traffic Manager "service", as there is no actual IP for ATM - it's for the Probe IPs specifically, which are documented here: https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-faqs.

azurecla commented:
Hi __@KundanaP__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ryanbor committed:
NSG Service Tags: Fix AzureTrafficManager tag description

Modified the Azure Traffic Manager tag to properly reflect the behavior of the tag. It isn't for the Traffic Manager "service", as there is no actual IP for ATM - it's for the Probe IPs specifically, which are documented here: https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-faqs.
